src/daemon/control_protocol_impl.rs

```diff
@@ -45,8 +45,15 @@ impl ControlProtocolImpl {
 impl ControlProtocol for ControlProtocolImpl {
     async fn bind_n2r(&self, socket_id: String, anon_id: Option<String>, dock: Option<Dock>) {
         let anon_id = anon_id.map(|id| self.anon_identities.lock().get(&id));
-        let socket = Socket::bind_n2r(&self.ctx, anon_id, dock);
+        let socket = Socket::bind_n2r(&self.ctx, anon_id.clone(), dock);
         self.sockets.insert(socket_id, socket);
+
+        if let Some(aid) = anon_id {
+            log::info!(
+                "----------- Bound N2rSocket with ANOOOON FINGERPRING: {} ----------",
```

Comment:
Misspelled.

In general, instead of dumping things into the logs to look at. you should add more control protocol methods. For example, one control protocol method (say `socket_info`) can take a socket_id and dump its fingerprint and dock number.